sdk/pre-authorized-debit-v1/package.json

```diff
@@ -5,7 +5,7 @@
     "main": "dist/index.js",
     "types": "dist/index.d.ts",
     "scripts": {
-        "build": "yarn build:anchor-client && yarn run tsc",
+        "build": "yarn run tsc",
```

Comment:
Why did you remove this?

Reply:
Build should ideally just produce the compiled js files. The `build:anchor-client` has a dependency on parent folders (pathing to the idl file), which is a little weird to have in a build command.

Reply:
Okay makes sense